### ISSUE TITLE
Parser error

### ISSUE THREAD

**Issue by rzlourenco** [fenix-framework/fenix-framework#201]
```
Exception in thread "main" java.lang.Error: line 2:3: unexpected char: 0xFFFF
	at pt.ist.fenixframework.DomainModelParser.getDomainModel(DomainModelParser.java:54)
	at pt.ist.fenixframework.DmlCompiler.getDomainModel(DmlCompiler.java:56)
	at pt.ist.fenixframework.DmlCompiler.compile(DmlCompiler.java:41)
	at pt.ist.fenixframework.dml.maven.AbstractDmlCodeGeneratorMojo.execute(AbstractDmlCodeGeneratorMojo.java:166)
	at pt.ist.fenixframework.dml.maven.DmlCodeGeneratorMojo.execute(DmlCodeGeneratorMojo.java:101)
	at pt.ist.fenixframework.maven.FFDmlCodeGeneratorMojo.execute(FFDmlCodeGeneratorMojo.java:111)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:132)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:120)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:355)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:155)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:584)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:216)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:160)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: line 2:3: unexpected char: 0xFFFF
	at pt.ist.fenixframework.dml.DmlLexer.nextToken(DmlLexer.java:218)
	at antlr.TokenBuffer.fill(TokenBuffer.java:69)
	at antlr.TokenBuffer.LA(TokenBuffer.java:80)
	at antlr.LLkParser.LA(LLkParser.java:52)
	at pt.ist.fenixframework.dml.DmlParser.domainDefinitions(DmlParser.java:71)
	at pt.ist.fenixframework.DomainModelParser.getDomainModel(DomainModelParser.java:49)
	... 26 more
```

It seems that when you have a comment in the end of file that does not end in `\n` you get a parser error when compiling. Example file (in hex format):
```
0000000: 636c 6173 7320 5374 7566 663b 0a2f 2f    class Stuff;.//
```

This works, though:
```
0000000: 636c 6173 7320 5374 7566 663b 0a2f 2f0a  class Stuff;.//.
```

**Comment by jcarvalho** [comment]
Could you please submit a Pull Request, adding a test case to the [DomainModelParserTest](https://github.com/fenix-framework/fenix-framework/blob/develop/core/dml-compiler/dml/src/test/java/pt/ist/fenixframework/DomainModelParserTest.java), highlighting the problem?

This way we can make sure the problem is replicated.

**Comment by jcarvalho** [comment]
Fixed by 9d5817fe85fd2fc0821e79e67b4c62e634347755. Thanks for the report :)
